external/kinesis-asl/src/main/scala/org/apache/spark/streaming/kinesis/KinesisInputDStream.scala

```diff
@@ -268,6 +273,30 @@ object KinesisInputDStream {
       this
     }
 
+    /**
+     * Sets the AWS DynamoDB endpoint URL. Defaults to
+     * "https://dynamodb.us-east-1.amazonaws.com" if no custom value is specified
```

Comment:
From where does this default coming from?

Reply:
The KCL will attempt to connect to the basic dynamo instance in the configured region. The KinesisInputDStream's default for the region is us-east-1, so naturally the default dynamodb endpoint would be `dynamodb.us-east-1.amazonaws.com`. 

It is not explicitly typed anywhere though. I could update this to be more indicative of the value in the `KinesisReceiver` (an `Option`), and specify that the default is `None` if that is preferable.

Reply:
Maybe it's better to do it just like `DEFAULT_KINESIS_ENDPOINT_URL`.